hubblestack_nova_profiles/cis/distribution-independent-linux-level-1-all-v1-1-0.yaml

```diff
@@ -858,7 +875,7 @@ stat:
       - /etc/gshadow-:
           gid: 0
           group: root
-          mode: 640
+          mode: '000'
```

Comment:
@alias454 First off thanks for putting in this effort to the profiles.
I just did some spot checks and this popped out - permissions for this file are different in different distros, at least on my default installs:
On Centos7:
```
  File: ‘/etc/gshadow-’
  Size: 338       	Blocks: 8          IO Block: 4096   regular file
Device: fd00h/64768d	Inode: 4525226     Links: 1
Access: (0000/----------)  Uid: (    0/    root)   Gid: (    0/    root)
```
On Ubuntu 18:
```
  File: /etc/gshadow-
  Size: 561       	Blocks: 8          IO Block: 4096   regular file
Device: ca01h/51713d	Inode: 216         Links: 1
Access: (0640/-rw-r-----)  Uid: (    0/    root)   Gid: (   42/  shadow)
```
On Ubuntu 16:
```
  File: '/etc/gshadow-'
  Size: 881       	Blocks: 8          IO Block: 4096   regular file
Device: fd01h/64769d	Inode: 146982      Links: 1
Access: (0600/-rw-------)  Uid: (    0/    root)   Gid: (    0/    root)
```
Should we differentiate it with osfinger grain selector (right below the data line)? 

I let Colton to also chime in since this is a merge to master PR and this fix can be done in a subsequent PR.

Reply:
I must have checked the actual filesystem and not the benchmark. In the benchmark the audit recommends "Run the following command and verify Uid is 0/root, Gid is 0/root or
<gid>/shadow, and Access is 640 or more restrictive." Thanks for catching that.

Reply:
This is one of those checks that's difficult -- the "640 or more restrictive" is hard to encode with the current modules. I anticipate we'll fix this properly with my nova rewrite -- hard to justify improvements to the modules before that.

Reply:
In principle, having different osfinger globs and specifying different modes for different distros should work. Arguably the profile doesn't live up to its name as the checks are not completely distro indipendent, but having all of them in one file is a good benefit regardless and makes working with them easier.

Reply:
Approving this now, the fixes have to come in subsequent PRs.